en_US/security/authn/user_management.md

```diff
@@ -34,9 +34,19 @@ The endpoints for importing users into the corresponding chains are:
 * `/api/v5/gateway/{protocol}/authentication/import_users`
 * `/api/v5/gateway/{protocol}/listeners/{listener_id}/import_users`
 
-The accepted `filename` parameter should be a node-local path of a file with credentials.
+The request should be a multipart form-data POST.
+
+Example:
+
+```
+curl -v -u admin:public -X 'POST' \
+    -H 'Content-Type: multipart/form-data' \
+    -F 'filename=@/tmp/myusers.csv' \
+    'http://localhost:18083/api/v5/authentication/password_based%3Abuilt_in_database/import_users'
+```
 
 The following file formats (identified by file extension) are supported:
+
 * CSV (`{"filename": "some-filename.csv"}`:
```

Comment:
We should update this too, simply "filename is `some/file/name csv`"

Reply:
I'll remove this part.
The curl command example is good enough.
Providing a json object here is actually quite confusing